test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -551,5 +555,11 @@ def verify_welcome_settings_checklist(self, option:str, list: list):
     def verify_action_button_enabled(self, option:str):
         assert BaseElement(str(CommunityWelcomeScreenComponents.ADD_NEW_ITEM_BUTTON.value)).is_enabled
         button_title = get_obj(CommunityWelcomeScreenComponents.ADD_NEW_ITEM_BUTTON.value).text
-        verify_equals(option, str(button_title))            
+        verify_equals(option, str(button_title))
+    
+    def copy_community_private_key(self):
+        Button('mainWindow_Back_up_StatusButton').click()
```

Comment:
this string should be const as well :) like `COMMUNITY_CHANNEL_SAVE_OR_CREATE_BUTTON: str = "createOrEditCommunityChannelBtn_StatusButton"`